profile.d/npm.sh

```diff
@@ -0,0 +1,22 @@
+export PATH="$HOME/.npm-packages/bin:$PATH" # man 1 sh
+export NODE_PATH="$HOME/.npm-packages/lib/node_modules" # man 1 node
+export NPM_CONFIG_PREFIX="$HOME/.npm-packages" # man 7 npm-config
+                                               # NOT man 1 or man 3
+export N_PREFIX="$HOME/.npm-packages" # installs under ~/.npm-packages/n
+
+install_node_version() {
+  rm -rf "$HOME"/.npm-packages
```

Comment:
Probably? I think `n` has some way to list versions which I can make a cache with. Will tinker with later.